### ISSUE TITLE
give Hogg the plots he needs for the APOGEE meeting

### ISSUE THREAD

**Issue by davidwhogg** [andycasey/AnniesLasso#39]
Most of what he needs will be satisfied by #36 but he also wants something which shows the non-zero parameters as a function of Lambda.  Also something that shows 2-d projections of the 17-d space for the training set, and for a much larger set of stars (if we can!)

**Comment by davidwhogg** [comment]
This has a strict deadline of 2016 January 7.

**Comment by andycasey** [comment]
Yo! I'll be getting back to this tomorrow when I return to work.

**Comment by andycasey** [comment]
"but he also wants something which shows the non-zero parameters as a function of Lambda"

Do you mean the theta coefficient values at many Lambda for a few terms (to see them 'snap off')?

**Comment by andycasey** [comment]
I'm just going to post additional (somewhat-relevant) plots here just in case you want them.

Here are distributions of the training set (random ~90% of the labelled set) for the 17-label case. One plot just shows TEFF, LOGG, FE_H, the other shows all.

![training-set-3-of-17](https://cloud.githubusercontent.com/assets/504436/12095143/afd689ec-b304-11e5-9e45-b7b4ac7cd0ae.png)

The 17-label figure is [here](https://cloud.githubusercontent.com/assets/504436/12095493/e1e1c044-b306-11e5-8200-ff2996c21b76.png).

**Comment by andycasey** [comment]
Sparsity (cross-posted from #37):
![sparsity](https://cloud.githubusercontent.com/assets/504436/12149657/c651082a-b49c-11e5-9dd2-c957cff10df4.png)

**Comment by andycasey** [comment]
(Although we are using a different metric to select the best model now,... here is the test scalar for the test set -- a random 1/10th of stars in the labelled set that were excluded from the training set.)

![test-scalar](https://cloud.githubusercontent.com/assets/504436/12149732/445ddaae-b49d-11e5-8432-c7e11605b900.png)

**Comment by davidwhogg** [comment]
AWESOME.  What is the minimum test-scalar point?

Oh and what is the test scalar?

**Comment by andycasey** [comment]
The test scalar in that figure is the summed chi^2 between stacked spectra in the validation set, and the spectral model at the ASPCAP labels for those stars in the validation set.

(I just shot you an email with >a dozen plots)

**Comment by davidwhogg** [comment]
Have you made the same plot for a label accuracy scalar?

**Comment by andycasey** [comment]
yah. check yo inbox
